src/network/announcethread.py

```diff
@@ -34,7 +33,7 @@ def announceSelf():
         for connection in connectionpool.pool.udpSockets.values():
             if not connection.announcing:
                 continue
-            for stream in state.streamsInWhichIAmParticipating:
+            for stream in connectionpool.pool.streams:
```

Comment:
`steamsInWhichIAmParticipating` should be done separately from `state.shutdown`

Reply:
separated this in two diff commits

Reply:
What I meant was, this PR should only deal with `state.shutdown`. There should be a separate PR, which removes the variable `state.streamsInWhichIAmParticipating` and replaces it just with `pool.streams`.